driver-adapters-wasm/neon-cfpages-remix/deployment-logs.txt

```diff
@@ -0,0 +1,13 @@
+✨ Compiled Worker successfully
+
+[31m✘ [41;31m[[41;97mERROR[41;31m][0m [1mReceived a malformed response from the API[0m
+
+  <!DOCTYPE html>
+  <!--[if lt IE 7]> <html class="no-js ie6 oldie" lang="en-US"> <![endif]-->
+  <!--[if I... (length = 6458)
+
+  GET /accounts/3dc90bc6bf50e78795c6f70865397810/pages/projects/neon-cfpages-remix/upload-token -> 504 Gateway Time-out
+  
+  If you think this is a bug, please open an issue at: [4mhttps://github.com/cloudflare/workers-sdk/issues/new/choose[0m
```

Comment:
The output looks wrong probably because of the API incident but I think it would be better to have it in gitignore anyway?